mysite/static/css/project/project.css

```diff
@@ -144,3 +144,7 @@ body#project .tab_nav li, body#project .tab_nav h3 {
 body#project .tab_nav .ui-state-active h3 {
     background: white;
     }
+#button-container {
```

Comment:
Suggestion here:

Use `body#something` to scope this to make sure this style only applies on the page where it needs to.